<launch>
  <node pkg="odometry_publisher" type="odometry_publisher" name="odometry_publisher">
    <param name="nav_odometry_topic" value="/odometry" />
    <param name="wheel_speed_topic" value="/wheelSpeeds" />
    <param name="wheel_radius" value="0.2" />
    <param name="baseline" value="0.7" />
  </node>
</launch>
